kingfisher_scrapy/spiders/europe_dynamic_spider_base.py

```diff
@@ -0,0 +1,31 @@
+import datetime
+
+import scrapy
+
+from kingfisher_scrapy.base_spider import CompressedFileSpider
+from kingfisher_scrapy.util import components, handle_http_error, join
+
+
+class EuropeDynamicSpiderBase(CompressedFileSpider):
+    # SimpleSpider
+    data_type = 'record_package'
+    date_format = 'year-month'
+
+    def start_requests(self):
+        yield scrapy.Request(
+            self.url,
```

Comment:
```suggestion
            f'{self.base_url}/ocds/services/recordpackage/getrecordpackagelist',
```
I think each spider should set a `base_url = 'https://www.ghaneps.gov.gh'` for example, and then here we request the corresponding endpoint.